streams/src/test/java/org/apache/kafka/streams/kstream/internals/KTableFilterTest.java

```diff
@@ -193,32 +193,6 @@ private void doTestValueGetter(final StreamsBuilder builder,
         }
     }
 
-    @Test
-    public void testValueGetter() {
```

Comment:
This can be merged with queryable value getter below, ditto in other test classes.